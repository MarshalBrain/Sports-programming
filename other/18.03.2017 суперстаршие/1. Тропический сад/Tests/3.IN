3
4 -2.27
-4 2.27
2.2 4