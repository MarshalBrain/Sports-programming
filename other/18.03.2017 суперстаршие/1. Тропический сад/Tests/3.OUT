4.599
0.000 0.000